tests/FieldAlias.php

```diff
@@ -71,6 +71,10 @@ public function testLegacyOrderBy()
     public function testLegacyOrderByFunction()
     {
         $mapper = test_spot_mapper('SpotTest\Entity\Legacy');
+        // Currently it works with mysql only
+        if (!$mapper->connectionIs('mysql')) {
+            $this->markTestSkipped('Not supported in Sqlite nor Postgres.');
```

Comment:
Why is this MySQL only? Do SQLite and PostgreSQL have issues supporting this?

Reply:
They have issues supporting the ```WEEK()``` function. I only tested for mysql but it is indeed bad design.